covid19-etl/etl/download_gb_by_taxid.yaml

```diff
@@ -0,0 +1,11 @@
+---
+taxid: 694009
+email: briano@bioteam.net
```

Comment:
this repo is public, are you comfortable sharing this email address? you could leave it blank and add a comment instead - can this be any email address, or it is the email of someone who has access to controlled access data?

Reply:
@paulineribeyre There's a more fundamental issue, this PR no longer has the latest/best code. I'd rather we just not do the merge, give me a day or 2 and I'll get you more complete code. So, suggest you not accept this PR.

Reply:
I saw another PR from the fork repository, but now it looks like removed (the fork), so I closed that PR. But it has the same changes. https://github.com/uc-cdis/covid19-tools/pull/12